### PR TITLE
Add repo.yml

### PR DESCRIPTION
**Pull request by nazrhom**
Change-type: patch
Signed-off-by: Giovanni Garufi <giovanni@balena.io>

### CODE REVIEW

File: repo.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+type: electron 
+sentry:
+    org: balenaetcher
+    team: resinio
```
Comment (lurch) [inline repo.yml]: Not `balenaio` ? :wink:

Comment (thundron) [reply repo.yml]: That's the sentry team the projects are accessible to

Comment (thundron) [reply repo.yml]: That's a sentry internal config we haven't changed yet

**Comment by nazrhom** [comment]
@resin-ci retest

**Pull request merged**